components/x-audio/src/components/index.jsx

```diff
@@ -15,25 +17,24 @@ export const Audio = ({
 	onPauseClick,
 	onCloseClick,
 	title,
-	seriesName
+	seriesName,
+	currentTime,
+	duration
 }) => (
 	<div className={classNameMap('audio-player', `audio-player--${expanded ? 'expanded' : 'minimised'}`)}>
 		{expanded && <button className={classNameMap('audio-player__minimise-button')} title='minimize player'/>}
 		{expanded && <div className={classNameMap('audio-player__control-timeline')}><input style={{width: '100%'}} type='range'/></div>}
 		{expanded && <button className={classNameMap('audio-player__rewind')} title='rewind 30 seconds'/>}
 		{expanded && <button className={classNameMap('audio-player__forward')} title='forward 30 seconds'/>}
 		{expanded && <button className={classNameMap('audio-player__control-speed')} title='change play speed'>x1</button>}
-
+		{expanded && !loading && <div className={classNameMap('audio-player__info__current-time')}>{formatTime(currentTime)}</div>}
```

Comment:
Perhaps not for this PR but as the source order is being controlled by CSS Grid, we could arrange these lines into groups to potentially make them more readable.

Might it be easier to read by grouping the elements within the `expanded` block? e.g.

```jsx
{expanded && (
  <button className={classNameMap('audio-player__minimise-button')} title='minimize player'/>
  <div className={classNameMap('audio-player__control-timeline')}><input style={{width: '100%'}} type='range'/></div>
)
```

Reply:
@benbarnett Is this possible to do?
I got this error when I try to do this way. Am I missing something?

```
Adjacent JSX elements must be wrapped in an enclosing tag. Did you want a JSX fragment <>...</>?
```